### PR TITLE
[AKS] `az aks create/update`: Add support for updating tags

### PR DESCRIPTION
**Pull request by FumingZhang**
**Description**

Add support for updating tags of the cluster in `aks update`.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -2870,13 +2872,19 @@ def aks_update(cmd, client, resource_group_name, name,
...
+                       '"--tags"')
 
     if not enable_managed_identity and assign_identity:
         raise CLIError(
             '--assign-identity can only be specified when --enable-managed-identity is specified')
 
     instance = client.get(resource_group_name, name)
+
+    # update tags
+    if tags is not None:
```
Comment (andyliuliming) [inline src/azure-cli/azure/cli/command_modules/acs/custom.py:2885]: I think we may need to merge the tags with the current one.

**Review by andyliuliming — CHANGES_REQUESTED** [review]

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/custom.py (hunk 1 above)
Comment (FumingZhang) [reply src/azure-cli/azure/cli/command_modules/acs/custom.py:2885]: Confirming the expected behavior of this option with PM.

For now, if the user provides an empty string when specifying the `--tags` option, the original tags will be cleared. If the user does not explicitly specify the `--tags` option, the original tags will not be changed.

Comment (FumingZhang) [reply src/azure-cli/azure/cli/command_modules/acs/custom.py:2893]: Confirmed with PM, the replacement update is the expected behavior.

**Review by andyliuliming — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**